Airports/EGPE/Ownership.txt

```diff
@@ -1,25 +1,39 @@
 SECTOR:Only PETWR 1:0:0
-OWNER:PER:PEA:AET
+OWNER:PER:PET
 
 SECTOR:Only PETWR 2:0:0
-OWNER:PER:PEA:SN:SE:S:SM:O
+OWNER:PER:SN:SE:S:SM:O
 
 SECTOR:Only PEAPP:0:0
-OWNER:PER:PEA
+OWNER:PER
 
 SECTOR:PETWR:0:2000
-OWNER:PET:PER:PEA:SN:SE:S
-ALTOWNER:Observing Scottish FIR:O:SM:PET:PER:PEA:SN:SE:S
-ALTOWNER:Observing Scottish FIR (excl. STC_CTR):O:SM:PET:PER:PEA:SN:SE:S
-ALTOWNER:Mentoring TWR/APP:PETM:PET:PER:PEA:SN:SE:S
+OWNER:PET:PER:SN:SE:S
+ALTOWNER:Observing Scottish FIR:O:SM:PET:PER:SN:SE:S
+ALTOWNER:Observing Scottish FIR (excl. STC_CTR):O:SM:PET:PER:SN:SE:S
+ALTOWNER:Mentoring TWR/APP:PETM:PET:PER:SN:SE:S
 BORDER:PETWR
 ARRAPT:EGPE
 DEPAPT:EGPE
 
-SECTOR:PEAPP:0:11000
-OWNER:PER:PEA:SN:SE:S
-ALTOWNER:Observing Scottish FIR:O:SM:PER:PEA:SN:SE:S
-ALTOWNER:Observing Scottish FIR (excl. STC_CTR):O:SM:PER:PEA:SN:SE:S
-ALTOWNER:Mentoring TWR/APP:PEAM:PEA:SN:SE:S
+SECTOR:PEAPP:0:7000
+OWNER:PER:SN:SE:S
+ALTOWNER:Observing Scottish FIR:O:SM:PER:SN:SE:S
+ALTOWNER:Observing Scottish FIR (excl. STC_CTR):O:SM:PER:SN:SE:S
+ALTOWNER:Mentoring TWR/APP:PEM:SN:SE:S
 BORDER:PEAPP
-ARRAPT:EGPE
\ No newline at end of file
+ARRAPT:EGPE
```

Comment:
I think it would actually be useful to have 'pending strips' visible to APP even with TWR online. Added situational awareness!

```suggestion
ARRAPT:EGPE
DEPAPT:EGPE
```